### PR TITLE
[onert/gbs] Enable x86_64 gbs build

### PR DESCRIPTION
**Pull request by hseok-oh**
Enable x86_64 gbs build

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
I've rebased to latest master branch.

@chunseoklee @wateret PTAL

**Review by chunseoklee — APPROVED** [review]

File: infra/nnfw/cmake/buildtool/config/config_x86_64-tizen.cmake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#
+# aarch64 tizen compile options
+#
+
+message(STATUS "Building for AARCH64 Tizen")
+
+# Build flag for tizen
+set(CMAKE_C_FLAGS_DEBUG     "-O -g -DDEBUG")
+set(CMAKE_CXX_FLAGS_DEBUG   "-O -g -DDEBUG")
```
Comment (wateret) [inline infra/nnfw/cmake/buildtool/config/config_x86_64-tizen.cmake:9]: Is `-O` intended for debug build?

Comment (hseok-oh) [reply infra/nnfw/cmake/buildtool/config/config_x86_64-tizen.cmake:9]: Yes. tizen should use `-O`

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**